schemas/2016-06-01/Microsoft.RecoveryServices.json

```diff
@@ -141,120 +162,215 @@
             {
               "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
+          ],
+          "description": "Vault extended information."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.RecoveryServices/vaults/extendedInformation"
           ]
         }
       },
       "required": [
-        "type",
         "apiVersion",
-        "properties"
+        "name",
+        "properties",
+        "type"
       ],
       "description": "Microsoft.RecoveryServices/vaults/extendedInformation"
     }
   },
   "definitions": {
-    "Sku": {
+    "RawCertificateData": {
       "type": "object",
       "properties": {
-        "name": {
+        "authType": {
           "oneOf": [
             {
               "type": "string",
               "enum": [
-                "RS0"
+                "Invalid",
+                "ACS",
+                "AAD",
+                "AccessControlService",
+                "AzureActiveDirectory"
               ]
             },
             {
               "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
           ],
-          "description": "Gets or sets the sku name. Required for vault creation, optional for update. Possible values include: 'RS0'"
+          "description": "Specifies the authentication type."
         },
-        "tier": {
+        "certificate": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "integer"
+              }
```

Comment:
This doesn't look right. According to our REST API docs, this should be a string: https://docs.microsoft.com/en-us/rest/api/recoveryservices/VaultCertificates/Create#rawcertificatedata.

In the spec it's defined as a 'byte' format string:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/registeredidentities.json#L151-L155

@leni-msft would you mind taking a look into this? Seems like this may be a problem with https://github.com/Azure/autorest.azureresourceschema. In the [OpenAPI spec](https://swagger.io/docs/specification/data-models/data-types/), it's defined as a string with 'base64-encoded characters'.

Reply:
@anthony-c-martin probably a bug of schema extension. Raised a issue https://github.com/Azure/autorest.azureresourceschema/issues/60 to track this